7_react/react1/week2/TODO-LIST/src/App.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import toDoItems from "./component/TodoItems";
+import Timer from "./component/timer";
+import "./App.css";
+
+
+function App() {
+  const [toDoItem, setToDoItems] = useState(toDoItems);
```

Comment:
The name of the state variable should be `toDoItems`, if the name of the setter is `setToDoItems`. The reason why you did not do this is that the initial to-do array that you import is already called `toDoItems`. However, you can just call the imported initial to-dos as `initialToDoItems`, and then this line can be:

```javascript
const [toDoItems, setToDoItems] = useState(initialToDoItems);
```